src/influxdb_ioxd/http_routes.rs

```diff
@@ -98,6 +98,9 @@ pub enum ApplicationError {
     #[snafu(display("Invalid request body: {}", source))]
     InvalidRequestBody { source: serde_json::error::Error },
 
+    #[snafu(display("Invalid response body: {}", source))]
+    InvalidResponseBody { source: serde_json::error::Error },
```

Comment:
This seems like it would be more of an internal error (if IOx couldn't serialize a data structure it created itself that would signal a coding / logic error rather than something the user could do about it)

In that case I think we have been more or less following the convention of Prefixing the error variant and the message with `Internal` to signal it is unexpected. For example:


```suggestion
    #[snafu(display("Internal error generating response body: {}", source))]
    InternalSerializingResponse { source: serde_json::error::Error },
```

Reply:
yep; indeed the error code mapping below, this error maps to an internalservererror.

Perhaps `InternalSerializationError`?

Reply:
👍